### ISSUE TITLE
how to use NavigationDrawer click to replace the layout of materialviewpager

### ISSUE THREAD

**Issue by bilalilyas90** [florent37/MaterialViewPager#172]
Hi!
please can you help me how to use navigationdrawer so that I can have different  layouts for different drawer items.

Thanx in advance

**Comment by antoxa2584x** [comment]
I think thia is not issue, so you spam.

**Comment by bilalilyas90** [comment]
Yes I know ... I am asking to guide me how to use it bcoz it's confusing me how to use it ... how to change materialviewpager on drawe item click.
